test/cucumber/evaluation.feature

```diff
@@ -0,0 +1,29 @@
+Feature: vizualize evaluation
+	 As a teacher
+	 I want to see evaluation
+	 So I can see any details that i want
+
+
+Scenario: vizualize evaluation
+	Given I am on Teaching Assistant Teacher's home page
+	When i click on the Vizualize button
+	Then I should see the Vizualization Page
+	When i put the name of the "evaluation1" on the "search" field
+	And click on search
+	Then I should see all the evaluations that have the name writted
```

Comment:
melhor: Then I should see all the evaluations that contains "evaluation1" in the name